frontend/src/components/regions/CardDetailsModal/Reviews.js

```diff
@@ -86,7 +86,7 @@ export default ({ reviewIds, reviews }) => {
         <Table stickyHeader aria-label="sticky table">
           <TableHead>
             <TableRow>
-              <TableCell>Timestamp</TableCell>
+              <TableCell>timestamp</TableCell>
```

Comment:
Why are you correcting this? Aren't you meant to be completely replacing this?

Reply:
Please upgrade this file so that the existing stories draw the correct things

Reply:
This has also been amended

Reply:
Tables have been removed entirely